controllers/kafkauser_controller.go

```diff
@@ -238,7 +238,13 @@ func (r *KafkaUserReconciler) Reconcile(ctx context.Context, request reconcile.R
 				return requeueWithError(reqLogger, "failed to reconcile user secret", err)
 			}
 		}
-		kafkaUser = user.DN()
+		kafkaUser, err = user.DN()
+		if err != nil {
+			reqLogger.Error(err, "Could not get Distinguished Name from the generated TLS certificate")
```

Comment:
I believe we shouldn't capitalize the first letter on error logging based on our [error handling guidelines](https://github.com/banzaicloud/developer-guide/blob/master/docs/coding-style/error-handling-guide.md#starting-error-message-with-capital-letter) 
```suggestion
			reqLogger.Error(err, "could not get Distinguished Name from the generated TLS certificate")
```